platform/linux-generic/arch/arm/odp/api/abi/cpu_inlines.h

```diff
@@ -20,6 +20,11 @@ static inline void _odp_cpu_pause(void)
 	__asm volatile("isb" ::: "memory");
 }
 
+static inline void _odp_prefetch_l1i(const void *addr)
+{
+	(void)addr;
```

Comment:
Instead of void casts, could use `ODP_UNUSED`.

Reply:
I'd rather leave it as "void cast" since ODP_UNUSED requires dependency to API hint header file, which these arch specific cpu_inlines.h files do not currently have.